certified-connectors/Eagle/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2577 @@
+            },
+            "PIPE_PHASE": {
+              "type": "string",
+              "description": "PIPE_PHASE"
+            },
+            "APP_CONF": {
+              "type": "string",
+              "description": "APP_CONF"
+            },
+            "DUE_BY": {
+              "type": "string",
+              "description": "DUE_BY"
+            },
+            "DUE_TIME": {
+              "type": "string",
+              "description": "DUE_TIME"
+            },
+            "START_DATE": {
+              "type": "string",
+              "description": "START_DATE"
+            },
+            "START_TIME": {
+              "type": "string",
+              "description": "START_TIME"
+            },
+            "END_DATE": {
+              "type": "string",
+              "description": "END_DATE"
+            },
+            "END_TIME": {
+              "type": "string",
+              "description": "END_TIME"
+            },
+            "ACT_TIME": {
+              "type": "string",
+              "description": "ACT_TIME"
+            },
+            "BILL_TIME": {
+              "type": "string",
+              "description": "BILL_TIME"
+            },
+            "AUTHOR": {
+              "type": "string",
+              "description": "AUTHOR"
+            },
+            "HOURS": {
+              "type": "string",
+              "description": "HOURS"
+            },
+            "WORKCODE": {
+              "type": "string",
+              "description": "WORKCODE"
+            },
+            "MANAGER": {
+              "type": "string",
+              "description": "MANAGER"
+            },
+            "LOCATION": {
+              "type": "string",
+              "description": "LOCATION"
+            },
+            "ISNEW": {
+              "type": "string",
+              "description": "ISNEW"
+            },
+            "ISDELETED": {
+              "type": "string",
+              "description": "ISDELETED"
+            }
+          }
+        },
+        "headers": {
+          "Location": {
+            "description": "Location",
+            "type": "string"
+          }
+        }
+      },
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {},
+            "headers": {
+              "Location": {
+                "description": "Location",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "summary": "When a task is created or changed",
+        "operationId": "TaskCreated",
+        "x-ms-visibility": "important",
+        "x-ms-trigger": "single",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "URL_TYPE": {
+                  "type": "string",
+                  "description": "URL_TYPE",
```

Comment:
@doniyorniazov could you please check descriptions for triggers? Descriptions should look like valid sentences or words. If it's necessary and more comprehensible for the users to keep descriptions this way, then please ignore this comment.